src/server/pfs/cmds/cmds.go

```diff
@@ -310,6 +310,39 @@ $ pachctl list-commit foo master --from XXX
 	listCommit.Flags().IntVarP(&number, "number", "n", 0, "list only this many commits; if set to zero, list all commits")
 	rawFlag(listCommit)
 
+	printCommitIter := func(commitIter client.CommitInfoIterator) error {
+		if raw {
+			for {
+				commitInfo, err := commitIter.Next()
+				if err == io.EOF {
```

Comment:
Does `SubscribeCommit` ever return `io.EOF`? I thought the point of that command was to block forever which means that this command will block forever, and in the case of where `raw=false` it won't return anything at all since it'll never call `Flush`